packages/invoices/src/OutboundShipment/DetailView/OutboundServiceLineEdit/useDraftServiceLines.ts

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect } from 'react';
+import {
+  RecordPatch,
+  InvoiceLineNodeType,
+  generateUUID,
+} from '@openmsupply-client/common';
+import {
+  useDefaultServiceItem,
+  ItemRowFragment,
+} from '@openmsupply-client/system';
+import {
+  useOutboundFields,
+  useOutboundServiceLines,
+  useSaveOutboundLines,
+  OutboundLineFragment,
+} from '../../api';
+import { DraftOutboundLine } from './../../../types';
+
+const createDraftLine = ({
+  item,
+  invoiceId,
+  seed,
+}: {
+  item: ItemRowFragment;
+  invoiceId: string;
+  seed?: OutboundLineFragment;
+}): DraftOutboundLine => ({
+  __typename: 'InvoiceLineNode',
+  id: generateUUID(),
+  item,
+  invoiceId,
+  totalAfterTax: 0,
+  totalBeforeTax: 0,
+  numberOfPacks: 0,
+  packSize: 0,
+  sellPricePerPack: 0,
+  note: '',
+  type: InvoiceLineNodeType.Service,
+  isCreated: !seed,
+  isUpdated: false,
+  isDeleted: false,
+  ...seed,
+});
```

Comment:
originally I created a new "DraftServiceLine" so that I could use a simpler shape. Buut.. it seemed simpler to just reuse the existing type rather than adding more things